CHANGES.txt

```diff
@@ -1,6 +1,11 @@
 Simphony-Numerrin CHANGELOG
 ======================
 
+Release 0.1.1
+-------------
+
+* Use latest version of of simphony-common (>=0.2.0) (#8)
```

Comment:
This version requirement could be added to setup.py at https://github.com/simphony/simphony-numerrin/blob/master/setup.py#L34